### DISCUSSION TITLE
Help setting rtl_433 to detect water wmbus signals.

### DISCUSSION THREAD

**Discussion by alvaromemo** [merbanan/rtl_433#2597] [Q&A] [answered]
Thank you in advance. 

I get a usb dongle to capture T1 datagrams, from water meters transmitting at 434.475Mhz. 

![image](https://github.com/merbanan/rtl_433/assets/34141161/bc8ecf33-3fa6-4266-9858-b53f94cf5463)

Using a SDR software, i can see some data on that frecuency, but using

rtl_433 -f 433.475M -vvv 

It shows nothing. Can you help?

**Comment by zuckschwerdt** [comment, marked as answer]
Wireless M-Bus, Mode C&T, 100kbps has a minimum samplerate of 1.2 MHz (12 samples of 100kb/s). Try `-s 1200k`

**Reply by alvaromemo** [reply]
Thank you. It works!.

Any other recommendation to get more telegrams?. Maybe using some parameters regarding Gain.

![image](https://github.com/merbanan/rtl_433/assets/34141161/3e2ff50c-f33d-40b4-8c33-bca842d91614)

**Reply by zuckschwerdt** [reply]
For gain the usual `-Y autolevel` and to keep an eye on levels `-M level -M noise` is recommended.

**Reply by alvaromemo** [reply]
Perfect. 

![image](https://github.com/merbanan/rtl_433/assets/34141161/8b313ef2-1516-494e-b8e1-f2820d6bf629)
